tools/gen-random-bout/gen-random-bout.cpp

```diff
@@ -58,80 +69,186 @@ static void push_range(KTest *b, const char *name, unsigned value) {
   o->numBytes = 4;
   o->bytes = (unsigned char *)malloc(o->numBytes);
 
-  *(unsigned*)o->bytes = value;
+  *(unsigned *)o->bytes = value;
+}
+
+int create_stat(size_t size, struct stat64 *s) {
+  char *filename = tempnam("/tmp", NULL);
+  int fd;
+  unsigned char *buf;
+
+  if ((fd = open(filename, O_CREAT | O_RDWR, S_IRWXU | S_IRWXG | S_IRWXO)) < 0)
+    return -1;
+
+  if ((buf = (unsigned char *)malloc(size)) == NULL)
+    return -1;
+
+  if (write(fd, buf, size) != (int)size) {
+    free(buf);
+    return -1;
+  }
+
+  fstat64(fd, s);
+
+  close(fd);
+
+  unlink(filename);
+
+  free(buf);
+  free(filename);
+  return 0;
 }
 
 int main(int argc, char *argv[]) {
   unsigned i, narg;
   unsigned sym_stdout = 0;
+  unsigned stdin_size = 0;
+  int total_args = 0;
+  unsigned total_files = 0;
+  unsigned file_sizes[1024];
+  char **argv_copy;
 
   if (argc < 2) {
     fprintf(stderr, "Usage: %s <random-seed> <argument-types>\n", argv[0]);
-    fprintf(stderr, "       If <random-seed> is 0, time(NULL)*getpid() is used as a seed\n");
-    fprintf(stderr, "       <argument-types> are the ones accepted by KLEE: --sym-args, --sym-files etc.\n");
-    fprintf(stderr, "   Ex: %s 100 --sym-args 0 2 2 --sym-files 1 8\n", argv[0]);
+    fprintf(stderr, "       If <random-seed> is 0, time(NULL)*getpid() is used "
+                    "as a seed\n");
+    fprintf(stderr, "       <argument-types> are the ones accepted by KLEE: "
+                    "--sym-args, --sym-files etc.\n");
```

Comment:
Same here

Reply:
@MartinNowack Also here the output of Lines 116 need not be indented.